doc/develop/getting_started/installation_linux.rst

```diff
@@ -227,32 +227,31 @@ The Zephyr SDK supports the following target architectures:
 
 Follow these steps to install the Zephyr SDK:
 
-#. Download and verify the `Zephyr SDK bundle
-   <https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.16.4>`_:
+#. Download and verify the `Zephyr SDK bundle`_:
 
-   .. code-block:: bash
+   .. parsed-literal::
```

Comment:
Nice find! I actually wanted to implement substitutions for the SDK version a while back but couldn't find a way to have them work in code-blocks :)
One thing that I'd like to see is that the block still renders the same as other code blocks (ex. it should have a background ; ideally, syntax highlighting too, I guess) 

<img width="800" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/ed6a52ca-b60f-45a0-a87b-c7e93512a478">

Reply:
https://github.com/sphinx-doc/sphinx/issues/3306

Reply:
Yeah, it's a bit of a compromise, I've figured you'd probably follow up with a strategic CSS change.

Reply:
I don't want to block the rest of this PR, but would it make sense to keep the doc update for later? Ideally you'd open a separate PR with the doc commit there that we (I) can tackle in due course. Beyond the (arguably not critical) issue described in this comment, there is a more concerning one with... wait for it... the PDF file :)

<img width="729" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/2bab5d53-0fd2-4e3c-9098-03c0c4430e40">

Reply:
> the PDF file

I HAVE AN IDEA!

Reply:
...is there a way of making the `parsed-literal` not highlight? Vaguely remember reading something about it, I'll look a bit and if that's not feasible I can pull the doc change.

Reply:
> > the PDF file
> 
> I HAVE AN IDEA!

PLEASE KILL PDF ONCE AND FOR ALL